dataservice/api/common/enumerations.yaml

```diff
@@ -0,0 +1,165 @@
+## ------------ NOTES ----------- ##
+# 'Unknown' - contributor specified this in source data
+# 'Not Reported' - should be used as the default if no value is specified in
+# the column or if this column doesn't exist in source data
+
+## --------- Questions --------- ##
+# How should values be cased - lower case, proper/title case?
+# What should default value be if Not Reported in source data?
+
+study:
+    release_status:
+        choices:
+            - Pending # Not sure about the choices here
+        default: Pending
+
+study_file:
+    availability:
+        choices:
+            - Immediate Download
+            - Cold Storage
+        default: Immediate Download
+
+participant:
+    gender:
+        choices:
+            - Male
+            - Female
+            - Unknown
+            - Not Reported
```

Comment:
We really need to be consistent with missing/unknown across all fields. It would be best to centralize them otherwise I've seen inconsistency happen before. I like to be a bit more concrete if this covers all of the cases:

`Not Reported` - means that the value was not reported, either to the original data source or to the DRC, could potentially mean that follow up could occur to gather the value if it's meaningful
`Not Allowed to Collect` - means that the value cannot be collected due to privacy, consent, etc other issues, follow up cannot occur
`Not Applicable` -  in the current context the value isn't applicable (e.g. tumor location for a structural birth defect

Other ones to consider:
`Not Available` - means that follow up was tried but the value isn't available
`Other` - a nicer catchall than just `missing` or `unknown`, often good to couple with a note/reason

I can also see `Unknown` being useful when that's the value provided, perhaps an indication like `Reported Unknown`.